yarn.lock

```diff
@@ -6445,18 +6445,18 @@ __metadata:
     got: ^11.8.2
     hpagent: ^0.1.2
     js-yaml: ^4.1.0
-    monaco-page-objects: ^2.0.0
+    monaco-page-objects: ^2.0.1
     sanitize-filename: ^1.6.3
     selenium-webdriver: ^4.2.0
     targz: ^1.0.1
     unzip-stream: ^0.3.0
-    vsce: ^2.9.2
+    vsce: ^2.3.0
```

Comment:
@ganeshrn This is downgrading the version of vsce due to the fact that https://github.com/redhat-developer/vscode-extension-tester/pull/460 was never merged. I was expected the change to fail CI/CD run but it passed.